openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx

```diff
@@ -40,6 +40,11 @@ export const threadFilterList = [
     value: 'Conversation',
     icon: <SVGIcons alt="" icon={Icons.COMMENT_GREY} width="16px" />,
   },
+  {
+    name: 'Announcement',
+    value: 'Announcement',
+    icon: <SVGIcons alt="" icon={Icons.ANNOUNCEMENT_BLACK} width="16px" />,
```

Comment:
Can we provide the alt value for these icons?

Reply:
Done